shared/test/unit/mocks/mock_moz_intl.js

```diff
@@ -0,0 +1,63 @@
+(function(global) {
+'use strict';
+
+global.MockMozIntl = {
+  formatList: function(list) {
+    return Promise.resolve(list.join(', '));
+  },
+  DateTimeFormat: function(locales, options) {
+    var formatter = Intl.DateTimeFormat(locales, options);
+    return {
+      // we need this because mozIntl options extend Intl options
+      resolvedOptions() { return options; },
+      format: formatter.format.bind(formatter)
+    };
+  },
+  DurationFormat: function(locales, options) {
+    const type = options.type;
+
+    const patterns = {
+      'hms': 'hh:mm:ss',
+      'msS': 'mm:ss.SS'
+    };
+
+    var numFormatter = Intl.NumberFormat(locales, {
+      style: 'decimal',
+      useGrouping: false,
+      minimumIntegerDigits: 2
+    });
+
+    return Promise.resolve({
+      format: function(input) {
```

Comment:
`format` has a lot of implementation here for a mock.  Wouldn't it be better to simply return `input` so that code using this method doesn't break, and if you need to program a specific behavior for tests, use stubs?